tests/config/test_themes.py

```diff
@@ -7,7 +7,7 @@
 
 
 expected_complete_themes = {
-    'default', 'gruvbox', 'light', 'blue'
+    'zt_dark', 'gruvbox_dark', 'zt_light', 'zt_blue',
```

Comment:
Can we leave the name of `gruvbox` as it is? This will give error for users who use `gruvbox` via zuliprc.

Reply:
You should find that you can still use `gruvbox` via the theme aliases.